### DISCUSSION TITLE
Coverlet.msbuild 2.6.2 Compatibility with .NET 6

### DISCUSSION THREAD

**Discussion by trabadi84** [coverlet-coverage/coverlet#1421] [General]
Are there known concerns regarding the transition to .NET 6 with this version or later versions

**Comment by daveMueller** [comment]
For `coverlet.msbuild 2.6.2` it's hard to tell as the version is approximately 3 1/2 years old. But in general you can use `coverlet.msbuild` in .NET 6 projects without concerns. The `msbuild` driver suffers from some known issues you can find [here](https://github.com/coverlet-coverage/coverlet/blob/master/Documentation/KnownIssues.md). That's why we often recommend to use `coverlet.collector`.

**Comment by Bertk** [comment]
Please see  https://www.nuget.org/packages/coverlet.collector/6.0.4
